sinks/api/decoder.go

```diff
@@ -98,6 +98,8 @@ func (self *defaultDecoder) getContainerMetrics(container *source_api.Container,
 		return nil
 	}
 	labels[sinksV1Api.LabelContainerName.Key] = container.Name
+	labels[sinksV1Api.LabelImage.Key] = container.Image
```

Comment:
Can you add this in sinks/api/v1/decoder.go as well? The code is in a state of flux. Thats the reason for duplication.